frontend/src/styles/_uswds-theme-custom-styles.scss

```diff
@@ -257,6 +257,10 @@ i.e.
   background-color: color("mint-40");
 }
 
+.bg-mint-70 {
+  background-color: color("mint-70");
+}
+
```

Comment:
There are existing [USWDS utility classes for background colors](https://designsystem.digital.gov/utilities/color/#background-color). You can just use `.bg-mint-70`.